starfish/image/_filter/mean_high_pass.py

```diff
@@ -60,28 +50,24 @@ def add_arguments(cls, group_parser: argparse.ArgumentParser) -> None:
     @staticmethod
     def high_pass(image: np.ndarray, size: Number, rescale: bool=False) -> np.ndarray:
```

Comment:
you could legitimately accept `xr.DataArray`, I *think*, but I'm not 100% sure.  it will just return `np.ndarray`